app/assets/stylesheets/govuk_publishing_components/components/_contents-list.scss

```diff
@@ -28,7 +28,7 @@
 
 .gem-c-contents-list__link {
   .gem-c-contents-list__list-item--parent > & {
-    font-weight: bold;
+    font-weight: $govuk-font-weight-bold;
```

Comment:
I appreciate that you didn't write this whole structure, but can we take the opportunity to make it more readable? I think lines 29 to 35 can be replaced with this:

```
.gem-c-contents-list__link {
  @include govuk-template-link-focus-override;
}

.gem-c-contents-list__list-item--parent > .gem-c-contents-list__link {
  font-weight: $govuk-font-weight-bold;
}
```

Reply:
Fixed as suggested.